extra/mender-client-docker-addons/Dockerfile

```diff
@@ -30,8 +30,8 @@ RUN (cd /src/mender-configure-module && \
 RUN mkdir --parents /mender-install/etc/mender
 
 WORKDIR /src/mender
-RUN if [ -f CMakeLists.txt ]; then cmake -D MENDER_NO_BUILD=1 -S .; fi
-RUN DESTDIR=/mender-install make prefix=/usr install
+RUN cmake -D CMAKE_INSTALL_PREFIX:PATH=/usr -S .
```

Comment:
Q: What's the extra `PATH` suffix?

Reply:
It is the "type" of the variable, see [here](https://cmake.org/cmake/help/latest/prop_cache/TYPE.html
).. I guess it would work anyway as an string, but 1) this is what I found in stackoverflow and 2) better to be explicit than sorry